### PR TITLE
[Support] Add timestamp to availability tests.

### PR DESCRIPTION
**Pull request by alext**
## What

This makes it much easier to correlate any failures with CF logs etc.

## How to review

Code review. Run availability tests and verify output looks good.

## Who can review

Anyone but myself (and probably @dcarley whose commit I lifted this from)

### CODE REVIEW

File: platform-tests/src/availability/healthcheck_availability_during_deployment_test.go [hunk 1]
```diff
@@ -45,7 +45,8 @@ func errorRateThreshold(metrics *vegeta.Metrics, minimumTestDuration time.Durati
 	metrics.Close()
 
 	fmt.Printf(
-		" - Duration: %s, Requests: %d, Non 200 Requests: %d, Success Rate: %.2f%%\n",
+		"%s: Duration: %s, Requests: %d, Non 200 Requests: %d, Success Rate: %.2f%%\n",
```
Comment (dcarley) [inline platform-tests/src/availability/healthcheck_availability_during_deployment_test.go]: Something that bugged me is that the timestamps aren't always the same length which makes it harder to match the other fields across multiple lines:
```
2017-01-17 12:01:07.320338954 +0000 UTC: Duration: 6m49.99999862s, Requests: 4101, Non 200 Requests: 0, Success Rate: 100.00%
2017-01-17 12:01:12.320654329 +0000 UTC: Duration: 6m54.99999862s, Requests: 4151, Non 200 Requests: 0, Success Rate: 100.00%
2017-01-17 12:01:17.32094242 +0000 UTC: Duration: 6m59.99999862s, Requests: 4201, Non 200 Requests: 0, Success Rate: 100.00%
2017-01-17 12:01:22.32122625 +0000 UTC: Duration: 7m4.99999862s, Requests: 4251, Non 200 Requests: 0, Success Rate: 100.00%
2017-01-17 12:01:27.321503754 +0000 UTC: Duration: 7m9.99999862s, Requests: 4301, Non 200 Requests: 0, Success Rate: 100.00%
2017-01-17 12:01:32.321791137 +0000 UTC: Duration: 7m14.99999862s, Requests: 4351, Non 200 Requests: 0, Success Rate: 100.00%
```

File: platform-tests/src/availability/healthcheck_availability_during_deployment_test.go [hunk 2]
```diff
@@ -45,7 +45,8 @@ func errorRateThreshold(metrics *vegeta.Metrics, minimumTestDuration time.Durati
...
+		time.Now(),
 		metrics.Duration,
```
Comment (dcarley) [inline platform-tests/src/availability/healthcheck_availability_during_deployment_test.go]: I wonder if it's still worth including `Duration`?

File: platform-tests/src/availability/healthcheck_availability_during_deployment_test.go (hunk 1 above)
Comment (alext) [inline platform-tests/src/availability/healthcheck_availability_during_deployment_test.go]: I'll have a play with this further while I'm on support starting tomorrow.

**Comment by alext** [comment]
Closing for further refinements.

**Comment by alext** [comment]
I've reworked this to use (mostly) constant-width formatting for the output.

**Comment by mtekel** [comment]
Works well:
```
2017-01-23 11:26:45.84 UTC: Duration:   45.30s, Requests:   454, Non 200 Requests:   0, Success Rate: 100.00%
2017-01-23 11:26:50.86 UTC: Duration:   50.30s, Requests:   504, Non 200 Requests:   0, Success Rate: 100.00%
2017-01-23 11:26:55.88 UTC: Duration:   55.30s, Requests:   554, Non 200 Requests:   0, Success Rate: 100.00%
2017-01-23 11:27:00.96 UTC: Duration:   60.40s, Requests:   605, Non 200 Requests:   0, Success Rate: 100.00%
```
Thanks for the improvement, merging.

**Pull request merged**